README.md

```diff
@@ -54,7 +54,7 @@ If they don't know your conference they can't buy a ticket and come to it 😏.
 ### April
 
 * 4-6: [QCon London](https://qconlondon.com/) - London (UK)
-* 7-8: [SymfonyLive Paris 2022](https://live.symfony.com/2022-paris/) - Paris (France) <a href="https://live.symfony.com/2022-paris/cfp"><img alt="CFP SymfonyLive Paris 2022" src="https://img.shields.io/static/v1?label=CFP&message=until%2024-Jan-2022&color=green"> </a>
+* 7-8: [S🤞ymfonyLive Paris 2022](https://live.symfony.com/2022-paris/) - Paris (France) <a href="https://live.symfony.com/2022-paris/cfp"><img alt="CFP SymfonyLive Paris 2022" src="https://img.shields.io/static/v1?label=CFP&message=until%2024-Jan-2022&color=green"> </a>
```

Comment:
Le rajout du smiley qui croise les doigts dans le talk "Symfony Live", c'est une coquille ? ^^

Reply:
C'est grosse erreur, je corrige.

Reply:
Should be good.

Reply:
thx :)